lib/data_cycle/schedule_course_updates.rb

```diff
@@ -41,21 +43,32 @@ def enqueue_course_updates
     courses_to_update.each do |course|
       CourseDataUpdateWorker.update_course(course_id: course.id, queue: queue_for(course))
 
+      UpdateWikidataStatsWorker.new.perform(course) if wikidata(course)
```

Comment:
Do this in UpdateCourseStats, rather than here. `ScheduleCourseUpdates` is designed to just put all the update jobs for current courses into the right queue, rather than do any of the actual data updates.

Doing it this way would mean that this process would first add a course CourseDataUpdateWorker job to the queue, then (before that job runs) it would do a `UpdateWikidataStatsWorker` job without going through the queue, and then it would move on to the next course to put one job in the queue and do the other in the current context.

Putting it into UpdateCourseStats means the existing update logging scheme can capture details about how long the wikidata part of it took for each course, and that time will contribute to the total average time of updates for that course (and so, it will affect which queue it should get sorted into).